cmd/frontend/main.go

```diff
@@ -52,7 +52,7 @@ var (
 	directProxy = flag.Bool("direct_proxy", false, "if set to true, uses the module proxy referred to by this URL "+
 		"as a direct backend, bypassing the database")
 	bypassLicenseCheck = flag.Bool("bypass_license_check", false, "display all information, even for non-redistributable paths")
-	hostAddr           = flag.String("host", "localhost:8080", "Host address for the server")
+	hostAddr           = flag.String("host", "localhost:8088", "Host address for the server")
```

Comment:
Why change this? Basically, do not change the unrelated part to our main purpose